packages/lesswrong/lib/rateLimits/utils.ts

```diff
@@ -120,7 +122,7 @@ export function calculateRecentKarmaInfo(userId: string, allVotes: RecentVoteInf
 }
 
 function getRateLimitName (rateLimit: AutoRateLimit) {
-  return `${rateLimit.itemsPerTimeframe} ${rateLimit.actionType} per ${rateLimit.timeframeLength} ${rateLimit.timeframeUnit}`
+  return rateLimit.rateLimitName
 }
```

Comment:
@Will-Howard It looks like this is moderator-facing (for example you can see them on the right-hand side of each card [here](https://forum.effectivealtruism.org/admin/moderation)), so it's not just used for analytics. And this makes it such that when a mod rate-limits a user, that will appear as like `moderatorRateLimit` instead of the actual rate limit.

My guess is that at least LW wouldn't want us to change this. I don't think you use `getRateLimitName()` in your new code anyway, so maybe it's best for us to revert this function and use a different name instead of `rateLimitType`, to reduce confusion?